benchmark/async_hooks/http-server.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  method: ['trackingEnabled', 'trackingDisabled', 'trackingRemoved'],
```

Comment:
Not a blocking comment, but these aren't methods, so it's possibly kind of confusing to call them methods.